readme.md

```diff
@@ -0,0 +1,22 @@
+#Tunnetler#
+
+Is a revival of old-school computer game, called Tunneler (You can play this at: https://classicreload.com/tunneler.html). Major difference of Tunnetler in comparison to it's predecessor will be network multiplayer. We dont want to bring Tunneler from the dead, we want to enhance it. So there are possible any reinforcements/enhancements, which will make Tunnetler even more fun!
```

Comment:
You forgot to add "an"

s/x/y/ means "substitute x with y". I was suggesting keeping only the word "enhancements".